src/Sylius/Behat/Context/Api/Admin/ManagingShipmentsContext.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Tester\Exception\PendingException;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Service\SharedStorage;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ShipmentInterface;
+use Sylius\Component\Customer\Model\CustomerInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingShipmentsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    public function __construct(ApiClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @When I browse shipments
+     */
+    public function iBrowseShipments(): void
+    {
+        $this->client->index('shipments');
+    }
+
+    /**
+     * @Then I should see( only) :count shipment(s) in the list
+     * @Then I should see a single shipment in the list
+     */
+    public function iShouldSeeCountShipmentsInList(int $count = 1): void
```

Comment:
```suggestion
    public function iShouldSeeCountShipmentsInTheList(int $count = 1): void
```

Reply:
yes, but i don't see any value to adding articles to the function name, here and bottom :)